apps/web/src/services/iam/components/UserGroupManagementAddGroupInfo.vue

```diff
@@ -1,5 +1,37 @@
 <script lang="ts" setup>
+import { reactive, watchEffect } from 'vue';
+
 import { PFieldGroup, PTextInput } from '@cloudforet/mirinae';
+
+const emit = defineEmits(['update:values']);
+
+const state = reactive({
+    groupName: '',
+    description: '',
+});
+
+watchEffect(() => {
+    emit('update:values', {
+        groupName: state.groupName,
+        description: state.description,
+    });
+});
+
+// watch(() => state, (value) => {
+//     emit('update:value', {
+//         groupName: value.groupName,
+//         description: value.description,
+//     });
+// }, { deep: true, immediate: true });
+
+/* Components */
+const handleChangeGroupName = (value: string) => {
+    state.groupName = value;
+};
+
+const handleChangeDescription = (value: string) => {
+    state.description = value;
+};
```

Comment:
👀 Couldn't this be combined into one? Using the useFormValidator hook might be a good idea!